core/src/main/java/greencity/repository/SocialNetworkImageRepo.java

```diff
@@ -13,4 +18,26 @@
      * @return Optional of {@link SocialNetworkImage}
      */
     Optional<SocialNetworkImage> findByHostPath(String hostPath);
+
+    /**
+     * Find {@link SocialNetworkImage} by page.
+     *
+     * @param pageable pageable configuration.
+     * @return {@link Page}
+     * @author Orest Mamchuk
+     */
+    Page<SocialNetworkImage> findAll(Pageable pageable);
```

Comment:
Method `findAll(Pageable)` is already available in JpaRepository that this interface extends, so no need to introduce it here.

Reply:
I know this, however I noticed it in UserRepo 

![image](https://user-images.githubusercontent.com/68025190/94238062-737b9b80-ff18-11ea-9c33-baeff9e281e9.png)